paper/src/main/java/com/denizenscript/denizen/paper/events/PlayerChunkUnloadScriptEvent.java

```diff
@@ -0,0 +1,67 @@
+package com.denizenscript.denizen.paper.events;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.ChunkTag;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import io.papermc.paper.event.packet.PlayerChunkUnloadEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class PlayerChunkUnloadScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // player unloads chunk
+    //
+    // @Group Paper
+    //
+    // @Location true
+    //
+    // @Plugin Paper
+    //
+    // @Warning This event will fire *extremely* rapidly and almost guarantees lag. Use with maximum caution.
+    //
+    // @Triggers when a player unloads a chunk.
```

Comment:
According to the JD
> Is called when a [Player](https://jd.papermc.io/paper/1.21.4/org/bukkit/entity/Player.html) receives a chunk unload packet.
Should only be used for packet/clientside related stuff. Not intended for modifying server side.

Should probably mention that this is just for client-side related things (e.g. most `fakeX` features) and `Prefer <@link event> for most use-cases` with a link to the normal chunk unload event.